tools/mkimage/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine:latest
```

Comment:
Slight preference to use a specific alpine version. Just in case things change on the alpine side and something breaks mysteriously.